src/components/graduation/SubjectCard.tsx

```diff
@@ -0,0 +1,48 @@
+﻿import type { SubjectRowData } from "@/types/graduation";
+import InfoTag from "./InfoTag";
+
+function SubjectCard({
+  courseCode,
+  courseName,
+  classification,
+  credits,
+  grade,
+  year,
+  semester,
+  gradeLevel,
+}: SubjectRowData) {
+
+  const showDetails = year && semester && gradeLevel;
```

Comment:
```suggestion
  const isShowDetails = year && semester && gradeLevel;
```

boolean 값은 is 등을 붙인다면 더 알아보기 쉬워서 좋을 거 같습니당

Reply:
헉 API 작업을 미리 하다가 이미 수정해버렸어요 ㅋㅋㅋ 꼼꼼한 리뷰 감사합니다아 ㅠㅠㅠㅠ